### DISCUSSION TITLE
Shortcuts for 90° rotate in the editor

### DISCUSSION THREAD

**Discussion by cihe13375** [ppy/osu#24048] [Q&A] [answered]
Are there shortcuts for 90° rotate in the editor (i.e. Ctrl+< and Ctrl+> in stable)? I know there are buttons (that's good) but I still feel having keyboard shortcuts would be convenient. I searched the shortcuts available in the options menu but didn't find them.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/24059
